apple/RNGestureHandler.m

```diff
@@ -474,6 +474,11 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer
     return YES;
   }
 
+  if ([otherGestureRecognizer.view isKindOfClass:[UIScrollView class]] &&
+      [gestureRecognizer isKindOfClass:[RNDummyGestureRecognizer class]]) {
```

Comment:
This also will not be enough - it makes every dummy recognizer simultaneous with everything that's attached to a scroll view. Can you add a check to make sure they are attached to the same view?